[BUG] Upload fails with NLTK issue after update to 2.11.5

### Description

I just updated my paperless-ngx docker container to 2.11.5 from 2.11.0.
Since the update, I can no longer upload new documents.

During ingestion of the uploaded document, I am seeing the issue shown below.

I've found #2876 from some time ago, but for me, it doesn't seem to be dependent on the document, nor does it work with documents which were previously uploaded without issue.

### Steps to reproduce

1. Upload any PDF file, e.g. by dragging and dropping it

### Webserver logs

```bash
[2024-08-23 03:20:58,067] [ERROR] [paperless.tasks] ConsumeTaskPlugin failed: test_file.pdf: The following error occurred while storing document test_file.pdf after parsing:

**********************************************************************

  Resource [93mpunkt_tab[0m not found.

  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk

  >>> nltk.download('punkt_tab')

  [0m

  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:

    - PosixPath('/usr/share/nltk_data')

**********************************************************************

Traceback (most recent call last):

  File "/usr/local/lib/python3.11/site-packages/asgiref/sync.py", line 327, in main_wrap

    raise exc_info[1]

  File "/usr/src/paperless/src/documents/consumer.py", line 670, in run

    document_consumption_finished.send(

  File "/usr/local/lib/python3.11/site-packages/django/dispatch/dispatcher.py", line 176, in send

    return [

           ^

  File "/usr/local/lib/python3.11/site-packages/django/dispatch/dispatcher.py", line 177, in <listcomp>

    (receiver, receiver(signal=self, sender=sender, **named))

               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/usr/src/paperless/src/documents/signals/handlers.py", line 95, in set_correspondent

    potential_correspondents = matching.match_correspondents(document, classifier)

                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/usr/src/paperless/src/documents/matching.py", line 37, in match_correspondents

    pred_id = classifier.predict_correspondent(document.content) if classifier else None

              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/usr/src/paperless/src/documents/classifier.py", line 413, in predict_correspondent

    X = self.data_vectorizer.transform([self.preprocess_content(content)])

                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/usr/src/paperless/src/documents/classifier.py", line 386, in preprocess_content

    words: list[str] = word_tokenize(

                       ^^^^^^^^^^^^^^

  File "/usr/local/lib/python3.11/site-packages/nltk/tokenize/__init__.py", line 142, in word_tokenize

    sentences = [text] if preserve_line else sent_tokenize(text, language)

                                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/usr/local/lib/python3.11/site-packages/nltk/tokenize/__init__.py", line 119, in sent_tokenize

    tokenizer = _get_punkt_tokenizer(language)

                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/usr/local/lib/python3.11/site-packages/nltk/tokenize/__init__.py", line 105, in _get_punkt_tokenizer

    return PunktTokenizer(language)

           ^^^^^^^^^^^^^^^^^^^^^^^^

  File "/usr/local/lib/python3.11/site-packages/nltk/tokenize/punkt.py", line 1744, in __init__

    self.load_lang(lang)

  File "/usr/local/lib/python3.11/site-packages/nltk/tokenize/punkt.py", line 1749, in load_lang

    lang_dir = find(f"tokenizers/punkt_tab/{lang}/")

               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/usr/local/lib/python3.11/site-packages/nltk/data.py", line 579, in find

    raise LookupError(resource_not_found)

LookupError:

**********************************************************************

  Resource [93mpunkt_tab[0m not found.

  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk

  >>> nltk.download('punkt_tab')

  [0m

  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:

    - PosixPath('/usr/share/nltk_data')

**********************************************************************

The above exception was the direct cause of the following exception:

Traceback (most recent call last):

  File "/usr/src/paperless/src/documents/tasks.py", line 149, in consume_file

    msg = plugin.run()

          ^^^^^^^^^^^^

  File "/usr/src/paperless/src/documents/consumer.py", line 733, in run

    self._fail(

  File "/usr/src/paperless/src/documents/consumer.py", line 304, in _fail

    raise ConsumerError(f"{self.filename}: {log_message or message}") from exception

documents.consumer.ConsumerError: test_file.pdf: The following error occurred while storing document test_file.pdf after parsing:

**********************************************************************

  Resource [93mpunkt_tab[0m not found.

  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk

  >>> nltk.download('punkt_tab')

  [0m

  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:

    - PosixPath('/usr/share/nltk_data')

**********************************************************************
```


### Browser logs

_No response_

### Paperless-ngx version

2.11.5

### Host OS

NixOS 24.05

### Installation method

Docker - official image

### System status

```json
{
    "pngx_version": "2.11.5",
    "server_os": "Linux-6.8.1-x86_64-with-glibc2.36",
    "install_type": "docker",
    "storage": {
        "total": 202324283392,
        "available": 116536750080
    },
    "database": {
        "type": "sqlite",
        "url": "/config/db.sqlite3",
        "status": "OK",
        "error": null,
        "migration_status": {
            "latest_migration": "paperless_mail.0011_remove_mailrule_assign_tag_squashed_0024_alter_mailrule_name_and_more",
            "unapplied_migrations": []
        }
    },
    "tasks": {
        "redis_url": "redis://broker:6379",
        "redis_status": "OK",
        "redis_error": null,
        "celery_status": "OK",
        "index_status": "OK",
        "index_last_modified": "2024-08-23T00:00:03.216949+02:00",
        "index_error": null,
        "classifier_status": "OK",
        "classifier_last_trained": "2024-08-23T01:05:00.319679Z",
        "classifier_error": null
    }
}
```


### Browser

_No response_

### Configuration changes

_No response_

### Please confirm the following

- [X] I believe this issue is a bug that affects all users of Paperless-ngx, not something specific to my installation.
- [X] I have already searched for relevant existing issues and discussions before opening this report.
- [X] I have updated the title field above with a concise description.

--- a/Dockerfile
+++ b/Dockerfile
@@ -246,7 +246,7 @@ RUN --mount=type=cache,target=/root/.cache/pip/,id=pip-cache \
   && echo "Installing NLTK data" \
     && python3 -W ignore::RuntimeWarning -m nltk.downloader -d "/usr/share/nltk_data" snowball_data \
     && python3 -W ignore::RuntimeWarning -m nltk.downloader -d "/usr/share/nltk_data" stopwords \
-    && python3 -W ignore::RuntimeWarning -m nltk.downloader -d "/usr/share/nltk_data" punkt \
+    && python3 -W ignore::RuntimeWarning -m nltk.downloader -d "/usr/share/nltk_data" punkt_tab \
   && echo "Cleaning up image" \
     && apt-get --yes purge ${BUILD_PACKAGES} \
     && apt-get --yes autoremove --purge \

--- a/docs/changelog.md
+++ b/docs/changelog.md
@@ -1,5 +1,38 @@
 # Changelog
 
+## paperless-ngx 2.11.5
+
+### Bug Fixes
+
+- Fix: use JSON for update archive file auditlog entries [@shamoon](https://github.com/shamoon) ([#7503](https://github.com/paperless-ngx/paperless-ngx/pull/7503))
+- Fix: respect deskew / rotate pages from AppConfig if set [@shamoon](https://github.com/shamoon) ([#7501](https://github.com/paperless-ngx/paperless-ngx/pull/7501))
+
+### Dependencies
+
+<details>
+<summary>5 changes</summary>
+
+- Chore(deps): Bump the small-changes group across 1 directory with 6 updates [@dependabot](https://github.com/dependabot) ([#7502](https://github.com/paperless-ngx/paperless-ngx/pull/7502))
+- Chore(deps-dev): Bump the development group with 2 updates [@dependabot](https://github.com/dependabot) ([#7497](https://github.com/paperless-ngx/paperless-ngx/pull/7497))
+- Chore(deps-dev): Bump axios from 1.6.7 to 1.7.4 in /src-ui [@dependabot](https://github.com/dependabot) ([#7472](https://github.com/paperless-ngx/paperless-ngx/pull/7472))
+- Chore(deps-dev): Bump ruff from 0.5.6 to 0.5.7 in the development group [@dependabot](https://github.com/dependabot) ([#7457](https://github.com/paperless-ngx/paperless-ngx/pull/7457))
+- Chore(deps): Bump the small-changes group with 3 updates [@dependabot](https://github.com/dependabot) ([#7460](https://github.com/paperless-ngx/paperless-ngx/pull/7460))
+</details>
+
+### All App Changes
+
+<details>
+<summary>7 changes</summary>
+
+- Fix: use JSON for update archive file auditlog entries [@shamoon](https://github.com/shamoon) ([#7503](https://github.com/paperless-ngx/paperless-ngx/pull/7503))
+- Chore(deps): Bump the small-changes group across 1 directory with 6 updates [@dependabot](https://github.com/dependabot) ([#7502](https://github.com/paperless-ngx/paperless-ngx/pull/7502))
+- Fix: respect deskew / rotate pages from AppConfig if set [@shamoon](https://github.com/shamoon) ([#7501](https://github.com/paperless-ngx/paperless-ngx/pull/7501))
+- Chore(deps-dev): Bump the development group with 2 updates [@dependabot](https://github.com/dependabot) ([#7497](https://github.com/paperless-ngx/paperless-ngx/pull/7497))
+- Chore(deps-dev): Bump axios from 1.6.7 to 1.7.4 in /src-ui [@dependabot](https://github.com/dependabot) ([#7472](https://github.com/paperless-ngx/paperless-ngx/pull/7472))
+- Chore(deps-dev): Bump ruff from 0.5.6 to 0.5.7 in the development group [@dependabot](https://github.com/dependabot) ([#7457](https://github.com/paperless-ngx/paperless-ngx/pull/7457))
+- Chore(deps): Bump the small-changes group with 3 updates [@dependabot](https://github.com/dependabot) ([#7460](https://github.com/paperless-ngx/paperless-ngx/pull/7460))
+</details>
+
 ## paperless-ngx 2.11.4
 
 ### Bug Fixes

